docker/Dockerfile

```diff
@@ -0,0 +1,116 @@
+# node:16.13.0
+FROM --platform=linux/amd64 node@sha256:1d35d3b639b639fb55d955d271c1da97d9e730c7dc0b9bae352a63c50b7c12c1 AS frontend
+
+WORKDIR /app/oppia
+
+RUN apt-get update -y && apt-get upgrade -y \
+    curl \
+    git \
+    npm
+
+# installing packages from the package.json file
+COPY package.json .
+COPY scripts/linters/custom_eslint_checks ./scripts/linters/custom_eslint_checks
+
+RUN npm install -g yarn@1.22.15 --force
+# mathjs ESOCKETTIMEDOUT error fix -- https://stackoverflow.com/questions/55845756/esockettimedout-error-while-yarn-install-command
```

Comment:
URL is not required, just write the reason in brief.

Reply:
done